data/json/npcs/BGSS_talk.json

```diff
@@ -0,0 +1,922 @@
+    "type": "talk_topic",
+    "dynamic_line": "That guy started a frenzy. People were already panicked. There was an armed guy overseeing the transport, acting like a cop but really he was just some kid they'd handed a rifle to. He tried to calm things down, and I guess it actually worked for a bit, although the 'kill the infected' bunch were pretty goddamn freaked out and were clearly ready to jump the moment the granny with the cut on her arm started frothing at the mouth. They started acting up again when we got to the camp. That didn't go well for them. A few heavily armed soldiers dragged them away, and I never saw them again.",
+    "responses": [
+      { "text": "How did you get out?", 
+        "trial": { "type": "PERSUADE", "difficulty": 30, "mod": [ [ "BRAVERY", 1 ], [ "TRUST", 1 ] ] },
+        "success": { "topic": "BGSS_FEMA_EVACUEE_1_ESCAPE1" },
+        "failure": { "topic": "BGSS_FEMA_EVACUEE_1_NOTYET" }
+      },      
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+  },
+  {
+    "id": "BGSS_FEMA_EVACUEE_1_ESCAPE1",
+    "type": "talk_topic",
+    "dynamic_line": "That place was chaos. I only stayed a few hours. They had a big backhoe running, digging a huge pit in a cordoned section they wouldn't let us near. Well, I managed to sneak over that way, and saw them dumping load after load of the dead in the pit, pouring dirt back over them even as they revived and tried to climb out. Even with all the shit I've seen since, it haunts me. I knew then I had to get out. Luckily for me, we were attacked the next morning by some giant horror, a kind I haven't really seen since then. While the guards were busy with that, I grabbed some supplies I'd stocked up over the night and I fucked right out of there. A few others tried to fuck out with me, but as far as I know I was the only lucky one.",
+    "responses": [
+      { "text": "Tell me more about that FEMA camp.", "topic": "BGSS_FEMA_EVACUEE_1_STORY2" },    
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+  },
+  {
+    "id": "BGSS_FEMA_EVACUEE_1_NOTYET",
+    "type": "talk_topic",
+    "dynamic_line": "That's a story for another day. I don't really like thinking about it.",
+    "responses": [
+      { "text": "Sorry. Tell me more about that FEMA camp.", "topic": "BGSS_FEMA_EVACUEE_1_STORY2" },  
+      { "text": "Sorry for asking.", "topic": "TALK_FRIEND" }
+    ]
+  },
+  {
+    "id": "BGSS_LEFT_FOR_DEAD_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "I was late to evacuate when the shit hit the fan. Got stuck in town for a few days, survived by hiding in basements eating girl guide cookies and drinking warm root beer. Eventually I managed to weasel my way out without getting caught by the zombies. I spent a few days holed up in an abandoned mall, but I needed food so I headed out to fend for myself in the woods. I wasn't doing a great job of it, so I'm kinda glad you showed up.",
```

Comment:
replace girl guide with girl scout unless this survivor is not supposed to be from America.

Reply:
I suspect this will be a recurring issue, though I'm doing my best to remain culturally accurate

Reply:
It happens, and you're getting it right 99% of the time.